includes/backup.js

```diff
@@ -190,22 +217,110 @@ function processBatchSet(dbUrl, parallelism, log, batches, ee, start, grandtotal
             });
           }
         });
-        total += output.length;
-        var t = (new Date().getTime() - start) / 1000;
-        ee.emit('received', {length: output.length, time: t, total: total, data: output, batch: thisBatch}, q, logCompletedBatch);
+        callback(null, output);
+      } else {
+        callback(err);
+      }
+    });
+  }
+
+  // fetch docs seperately (not using the /_bulk_get API)
+  function doGet(docs, callback) {
+    var output = [];
+    var hasErrored = false;
+
+    var getDocQ = async.queue(function(doc, done) {
+      if (!doc || hasErrored) return done();
+
+      var req = {
+        url: dbUrl + '/' + encodeURI(doc.id),
+        qs: { rev: doc.rev, revs: true },
```

Comment:
I think this request is going to fail if the `_rev` specified is deleted. I'm guessing this is why the deleted flag was added to the log file. I think there needs to be more testing around this sort of edge case.